src/Container/AbstractObjectContainer.php

```diff
@@ -27,43 +27,43 @@ public function offsetExists($offset): bool
     }
 
     #[\ReturnTypeWillChange]
-    public function offsetGet($offset)
+    public function offsetGet($offset): mixed
     {
         $element = array_slice($this->objects, $offset, 1);
 
         return $this->offsetExists($offset) ? array_shift($element) : null;
     }
 
     #[\ReturnTypeWillChange]
-    public function offsetSet($offset, $value)
+    public function offsetSet($offset, $value): mixed
```

Comment:
`mixed` is available only since PHP >= 8.0, the project is usable from PHP >= 7.4